.github/workflows/main.yml

```diff
@@ -119,7 +119,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10"]
-        os: [ubuntu-latest]
```

Comment:
Is there a reason you only pinned this one?  We point to `ubuntu-latest` in multiple places in this file.